code/_onclick/hud/parallax.dm

```diff
@@ -2,7 +2,7 @@
 /datum/hud/proc/create_parallax(mob/viewmob)
 	var/mob/screenmob = viewmob || mymob
 	var/client/C = screenmob.client
-	if (!apply_parallax_pref(viewmob)) //don't want shit computers to crash when specing someone with insane parallax, so use the viewer's pref
+	if (!apply_parallax_pref(viewmob) || is_ground_level(viewmob.z)) //don't want shit computers to crash when specing someone with insane parallax, so use the viewer's pref
```

Comment:
this shouldnt be needed inside create paralla itself imo

Reply:
yes